distros/generate.sh

```diff
@@ -9,7 +9,13 @@ set -e
 #	or: ./generate.sh debian-newversion
 #		to create a new folder and a Dockerfile within it
 
-cd "$(dirname "$(readlink -f "$BASH_SOURCE")")"
+# `readlink -f` does not work on macOS.
+# To get `greadlink`, run `brew install coreutils`.
+if [ "$(uname)" == "Darwin" ]; then
```

Comment:
Remember that this requires an install while changing `seq 1` to `seq 1 1` does not.